### PR TITLE
fix the download links

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

**Comment by DanielRuf** [comment]
Bump.

File: doc/community.html [hunk 1]
```diff
@@ -16,8 +16,7 @@
 <h2>Source code</h2>
 <p>
 The source code is on github at <a href="https://github.com/SOASTA/boomerang">github.com/SOASTA/boomerang</a>.  Feel
-free to fork it and contribute to it.  You can also get a <a href="http://github.com/SOASTA/boomerang/archives/master">tarball
-or zip archive of the code</a>.
+free to fork it and contribute to it. You can also get a <a href="https://github.com/SOASTA/boomerang/archive/master.tar.gz">tarball</a> or a <a href="github.com/SOASTA/boomerang/archive/master.zip">zip</a> of the code</a>.
```
Comment (bluesmoon) [inline doc/community.html]: The link to the zip file needs `https://` before it.

Comment (bluesmoon) [reply doc/community.html]: also, please split this onto multiple lines

File: doc/index.html [hunk 2]
```diff
@@ -40,7 +40,7 @@
 </p>
 <h2>where?</h2>
 <ul>
-<li>Get a zip or tarball from <a href="http://github.com/SOASTA/boomerang/archives/master">github.com/SOASTA/boomerang/archives/master</a></li>
+<li>Get a zip from <a href="https://github.com/SOASTA/boomerang/archive/master.zip">github.com/SOASTA/boomerang/archive/master.zip</a> or a tarball from <a href="github.com/SOASTA/boomerang/archive/master.tar.gz">https://github.com/SOASTA/boomerang/archive/master.tar.gz</a></li>
```
Comment (bluesmoon) [inline doc/index.html]: link to the tar.gz needs https:// before it

**Comment by bluesmoon** [comment]
Leave the base branch as `SOASTA:master` since we merge master into `gh-pages`

**Comment by DanielRuf** [comment]
Well, seems I have to recreate it or rebase it then.

**Comment by bluesmoon** [comment]
ok, let me see what I can do.

**Comment by DanielRuf** [comment]
That would eb good, you can also push to my branch.

**Comment by bluesmoon** [comment]
ok, so the issue is that we've moved to using markdown. I made the fix to the markdown files and pushed it.  Let me know if it looks okay: https://soasta.github.io/boomerang/doc/

**Comment by DanielRuf** [comment]
It should be `archive` instead of `archives` regarding the zip download.

**Comment by bluesmoon** [comment]
should now be fixed.

**Pull request closed without merging**